docs/developer-docs/latest/setup-deployment-guides/configurations/optional/sso.md

```diff
@@ -1,11 +1,9 @@
 ---
 title: Single Sign-On (SSO) - Strapi Developer Documentation
-description: 
+description: Strapi's Single Sign-On allows you to configure additional sign-in and sign-up methods for your administration panel. It requires an Entreprise Edition with a Gold plan.
```

Comment:
This description is too long, it would be nice to not exceed 160 characters :)

Reply:
```suggestion
Strapi's SSO allows you to configure additional sign-in and sign-up methods for your administration panel. It requires an Entreprise Edition with a Gold plan.
```